packages/dashboards/tests/integration/components/navi-dashboard-test.js

```diff
@@ -115,4 +116,36 @@ test('dashboard export', function(assert) {
 
     config.navi.FEATURES.enableDashboardExport = originalFeatureFlag;
   });
-})
\ No newline at end of file
+});
+
+test('dashboard schedule - config', function (assert) {
+  assert.expect(2);
+  let originalFeatureFlag = config.navi.FEATURES.enableScheduleDashboards;
+
+  return wait().then(() => {
```

Comment:
Looks like `beforeEach` is returning a promise. Is this `wait()` still needed?

Reply:
It is needed.

Reply:
Oh ok. i missed the setting of the feature flag. Thats probably why this wait is needed.